core/src/main/kotlin/org/evomaster/core/problem/graphql/GraphQLUtils.kt

```diff
@@ -175,79 +174,62 @@ object GraphQLUtils {
 
     }
 
-    fun getPrintableInputGene(inputGenes: List<Gene>, targetFormat: OutputFormat? = null): MutableList<String> {
+    private fun getPrintableInputGene(inputGenes: List<Gene>, targetFormat: OutputFormat? = null): MutableList<String> {
         val printableInputGene = mutableListOf<String>()
 
         for (gene in inputGenes) {
-            if  (gene.getWrappedGene(EnumGene::class.java)!=null) {//enum gene
-                //if it is optional it should be active
-                if ((gene.getWrappedGene(OptionalGene::class.java)?.isActive == true)||(gene.getWrappedGene(OptionalGene::class.java) == null)) {
+            //if it is opt , it should be active
+            //if(gene.getWrappedGene(OptionalGene::class.java)?.isActive != false)
+            if ((gene.getWrappedGene(OptionalGene::class.java)?.isActive == true) || (gene.getWrappedGene(OptionalGene::class.java) == null))
+                printableInputGene.add(inputsPrinting(gene, targetFormat))
 
-                    val i = gene.getValueAsRawString()
-                    printableInputGene.add("${gene.name} : $i")
-
-                }
+        }
+        return printableInputGene
+    }
 
+    fun inputsPrinting(
```

Comment:
add docs stating what this function is supposed to do (or provide better function name)

Reply:
Done